README.md

```diff
@@ -781,7 +785,11 @@ Response:
 		"title": "decompress/unpack",
 		"app_weight": 1,
 		"file_type": "CompressedFileFormatArchive",
-	}]]
+	}]],
+    "fileinfo": [
+        {"prefix": "file1.txt", "suffix": null},
```

Comment:
Can we get an example on something with a double suffix, file.fasta.gz ?

Reply:
done